src/lib/plugins/world.js

```diff
@@ -143,6 +150,10 @@ module.exports.server = async function (serv, { version, worldFolder, generation
 }
 
 module.exports.player = function (player, serv, settings) {
+  player.save = () => {
```

Comment:
refactor playerDat.save -> player.save to expose it

Reply:
For any added function please add to doc